Update to the available redundancy options explanation (#128388)
Azure/azure-content: articles/storage/common/storage-account-overview.md

<sup>1</sup> Data Lake Storage is a set of capabilities dedicated to big data analytics, built on Blob Storage. For more information, see [Introduction to Data Lake Storage](../blobs/data-lake-storage-introduction.md) and [Create a storage account to use with Data Lake Storage](../blobs/create-data-lake-storage-account.md).

> <sup>2</sup> ZRS, GZRS, and RA-GZRS are not available for all storage account types. They are supported for standard general-purpose v2, premium block blob, premium file share, and premium page blob accounts; however, the available redundancy options vary depending on the account type and the region. For more information, see [Azure Storage redundancy](storage-redundancy.md).

Reviewer: ```suggestion
<sup>2</sup> ZRS, GZRS, and RA-GZRS are not available for all storage account types. They are supported for standard general-purpose v2, premium block blob, premium file share, and premium page blob accounts. However, the available redundancy options can vary not only by type, but also by region. For more information, see [Azure Storage redundancy](storage-redundancy.md).
```

Reviewer: Hi @ivanzhang1992 - Thanks for the contribution. I have a suggestion here for greater clarity. Since the account types are already listed in the previous sentence, can we just add that availability can vary also by region?

Reviewer 2: @ivanzhang1992 Can you respond to the requested changes or let us know if you would like us to commit them?

@normesta - When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

Author: We need to note that Azure Files does not support RA-GRS. So, the redundancy option depends not only on the region, but also on the service type of the storage account. That’s why I used “region and type.”

Reviewer: Thanks @ivanzhang1992. Aren't the supported account types clearly itemized in the previous sentence? Adding "however" as if to say that the "type" mentioned in that next sentence is somehow additive is confusing. If you want "However", then the sentence should convey that there is no other additive type. I modified my previous edit with that variation. If you want to specifically call out that Azure Files is not supported, perhaps you could just keep the previous construction and then add that clarification. So simply "Azure Files is not yet supported".

Author: Thanks @normesta 
You’re right. I think we can commit this change.
「However, the available redundancy options can vary not only by type, but also by region. 」
